components/ratings/ratings.lua

```diff
@@ -0,0 +1,423 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Ratings
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Arguments = require('Module:Arguments')
+local Array = require('Module:Array')
+local Date = require('Module:DateExt')
+local FnUtil = require('Module:FnUtil')
+local Json = require('Module:Json')
+local Lpdb = require('Module:Lpdb')
+local Table = require('Module:Table')
+local String = require('Module:StringUtils')
+
+--- Liquipedia Ratings (LPR)
+local Ratings = {}
+
+-- Parameters related to LPDB
+local STATIC_CONDITIONS_MATCH = '[[mode::3v3]] AND [[finished::1]] AND [[liquipediatiertype::!School]]'
```

Comment:
the 3v3 mode seems odly specific for a commons module^^

Reply:
Yes, it's explained (slightly) in the OP